CS4850Group5A/VirtuDoc
# Screenshot captures during GHA tests

# Summary
- Added screenshot capture helper methods
- Added automatic artifact upload at the end of the test suite

Closes #54

# How to Test
- Verify that all JUnit tests pass in a comment below from the GitHub actions bot.
- Open the GHA test details and verify that there is a `.zip` available for download that has screenshots of our website.
- Run the JUnit tests locally and confirm that `target/screenshots/` contains screenshots of the running application.

# Comments
- Screenshot archives are automatically deleted after 4 days, as there is a 1 GB limit to artifacts from GHA builds.
- If you want to add your own page to the screenshot collection, add the URL to the `@ValueSource` annotation in `src/test/java/com/virtudoc/web/E2EWalkthroughTest.java`.
